<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="DataGridViewDataErrorEventArgs.xml" source-language="en-US" target-language="pt-BR">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac507424dd07417db23cdbec88dd22f5097f25643bc.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">07424dd07417db23cdbec88dd22f5097f25643bc</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>Provides data for the <ph id="ph1">&lt;see cref="E:System.Windows.Forms.DataGridView.DataError" /&gt;</ph> event.</source>
          <target state="translated">Fornece dados para o evento de <ph id="ph1">&lt;see cref="E:System.Windows.Forms.DataGridView.DataError" /&gt;</ph> .</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>Handling the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridView.DataError&gt;</ph> event lets you handle exceptions thrown by code outside your control (for example, by an external data source).</source>
          <target state="translated">Manipulando o <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridView.DataError&gt;</ph> evento permite manipular exceções lançadas por código fora de seu controle (por exemplo, por uma fonte de dados externa).</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>Use the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context%2A&gt;</ph> property to determine the state of the <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> at the time of the exception.</source>
          <target state="translated">Use o <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context%2A&gt;</ph> propriedade para determinar o estado da <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> no momento da exceção.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>Use the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A&gt;</ph> property to retrieve the exception data.</source>
          <target state="translated">Use o <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A&gt;</ph> propriedade para recuperar os dados de exceção.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>If you want to handle the exception by additional event handlers, set the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A&gt;</ph> property to <ph id="ph2">`true`</ph>.</source>
          <target state="translated">Se você desejar tratar a exceção por manipuladores de eventos adicionais, defina o <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A&gt;</ph> propriedade <ph id="ph2">`true`</ph>.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>The <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.ColumnIndex%2A&gt;</ph> and <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.RowIndex%2A&gt;</ph> properties normally indicate the cell in which the data error occurred.</source>
          <target state="translated">O <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.ColumnIndex%2A&gt;</ph> e <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.RowIndex%2A&gt;</ph> propriedades normalmente indicam a célula na qual ocorreu o erro de dados.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>When the error occurs in an external data source, however, the data source may not provide the column in which the error occurred.</source>
          <target state="translated">Quando o erro ocorre em uma fonte de dados externa, no entanto, a fonte de dados pode não fornecer a coluna na qual ocorreu o erro.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>In this case, the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.ColumnIndex%2A&gt;</ph> property typically indicates the column of the current cell at the time of the error.</source>
          <target state="translated">Nesse caso, o <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.ColumnIndex%2A&gt;</ph> propriedade geralmente indica a coluna da célula atual no momento do erro.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>The following code example demonstrates how to respond to information provided by the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs&gt;</ph> class.</source>
          <target state="translated">O exemplo de código a seguir demonstra como responder às informações fornecidas pelo <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs&gt;</ph> classe.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>This example is part of a larger example available in the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> class overview topic.</source>
          <target state="translated">Este exemplo é parte de um exemplo maior disponível no <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> tópico de visão geral da classe.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.Windows.Forms.DataGridViewDataErrorEventArgs.#ctor(System.Exception,System.Int32,System.Int32,System.Windows.Forms.DataGridViewDataErrorContexts)">
          <source>The exception that occurred.</source>
          <target state="translated">A exceção que ocorreu.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="M:System.Windows.Forms.DataGridViewDataErrorEventArgs.#ctor(System.Exception,System.Int32,System.Int32,System.Windows.Forms.DataGridViewDataErrorContexts)">
          <source>The column index of the cell that raised the <ph id="ph1">&lt;see cref="E:System.Windows.Forms.DataGridView.DataError" /&gt;</ph>.</source>
          <target state="translated">O índice da coluna da célula que disparou o <ph id="ph1">&lt;see cref="E:System.Windows.Forms.DataGridView.DataError" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" uid="M:System.Windows.Forms.DataGridViewDataErrorEventArgs.#ctor(System.Exception,System.Int32,System.Int32,System.Windows.Forms.DataGridViewDataErrorContexts)">
          <source>The row index of the cell that raised the <ph id="ph1">&lt;see cref="E:System.Windows.Forms.DataGridView.DataError" /&gt;</ph>.</source>
          <target state="translated">O índice da linha da célula que disparou o <ph id="ph1">&lt;see cref="E:System.Windows.Forms.DataGridView.DataError" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" uid="M:System.Windows.Forms.DataGridViewDataErrorEventArgs.#ctor(System.Exception,System.Int32,System.Int32,System.Windows.Forms.DataGridViewDataErrorContexts)">
          <source>A bitwise combination of <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorContexts" /&gt;</ph> values indicating the context in which the error occurred.</source>
          <target state="translated">Uma combinação bit a bit de valores <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorContexts" /&gt;</ph> que indica o contexto no qual ocorreu o erro.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="M:System.Windows.Forms.DataGridViewDataErrorEventArgs.#ctor(System.Exception,System.Int32,System.Int32,System.Windows.Forms.DataGridViewDataErrorContexts)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorEventArgs" /&gt;</ph> class.</source>
          <target state="translated">Inicializa uma nova instância da classe <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorEventArgs" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>Gets details about the state of the <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridView" /&gt;</ph> when the error occurred.</source>
          <target state="translated">Obtém detalhes sobre o estado do <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridView" /&gt;</ph> quando o erro ocorreu.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>A bitwise combination of the <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorContexts" /&gt;</ph> values that specifies the context in which the error occurred.</source>
          <target state="translated">Uma combinação bit a bit dos valores <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorContexts" /&gt;</ph> que especifica o contexto no qual ocorreu o erro.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>Members of the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorContexts&gt;</ph> enumeration may be combined using the bitwise <ph id="ph2">`OR`</ph> operator to represent the state of a data-bound <ph id="ph3">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> when a data error occurred.</source>
          <target state="translated">Membros de <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorContexts&gt;</ph> enumeração pode ser combinada com o bit a bit <ph id="ph2">`OR`</ph> operador para representar o estado de uma associação de dados <ph id="ph3">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> quando ocorreu um erro de dados.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>For example, if a user enters an invalid cell value (such as entering a name in a cell that requires a date) and then selects a different cell, the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> will try to commit the invalid cell value.</source>
          <target state="translated">Por exemplo, se um usuário inserir um valor de célula inválida (como inserir um nome em uma célula que requer uma data) e, em seguida, seleciona uma célula diferente, o <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> tenta confirmar o valor de célula inválida.</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>When the commit fails, the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> will raise a <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridView.DataError&gt;</ph> event whose <ph id="ph3">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context%2A&gt;</ph> property will have a value of <ph id="ph4">&lt;xref:System.Windows.Forms.DataGridViewDataErrorContexts.Commit&gt;</ph> and <ph id="ph5">&lt;xref:System.Windows.Forms.DataGridViewDataErrorContexts.CurrentCellChange&gt;</ph>.</source>
          <target state="translated">Quando a confirmação falha, o <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> gerará um <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridView.DataError&gt;</ph> evento cujo <ph id="ph3">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context%2A&gt;</ph> propriedade terá um valor de <ph id="ph4">&lt;xref:System.Windows.Forms.DataGridViewDataErrorContexts.Commit&gt;</ph> e <ph id="ph5">&lt;xref:System.Windows.Forms.DataGridViewDataErrorContexts.CurrentCellChange&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>The following code example demonstrates how to investigate the error context.</source>
          <target state="translated">O exemplo de código a seguir demonstra como investigar o contexto do erro.</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>This example is part of a larger example available in the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> class overview topic.</source>
          <target state="translated">Este exemplo é parte de um exemplo maior disponível no <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> tópico de visão geral da classe.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception">
          <source>Gets the exception that represents the error.</source>
          <target state="translated">Obtém a exceção que representa o erro.</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception">
          <source>An <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> that represents the error.</source>
          <target state="translated">Um <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> que representa o erro.</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception">
          <source>To handle the exception by more than one <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventHandler&gt;</ph>, throw the exception again at the end of a handler by setting the <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A&gt;</ph> property to <ph id="ph3">`true`</ph>.</source>
          <target state="translated">Para lidar com a exceção por mais de um <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventHandler&gt;</ph>, lançar a exceção novamente no final de um manipulador, definindo o <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A&gt;</ph> propriedade <ph id="ph3">`true`</ph>.</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception">
          <source>The following code example demonstrates checking the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A&gt;</ph> property.</source>
          <target state="translated">O exemplo de código a seguir demonstra como verificar o <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A&gt;</ph> propriedade.</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception">
          <source>This example is part of a larger example available in the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> class overview topic.</source>
          <target state="translated">Este exemplo é parte de um exemplo maior disponível no <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> tópico de visão geral da classe.</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>Gets or sets a value indicating whether to throw the exception after the <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorEventHandler" /&gt;</ph> delegate is finished with it.</source>
          <target state="translated">Obtém ou define um valor que indica se uma exceção deve ser gerada após o delegado <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorEventHandler" /&gt;</ph> terminar com ele.</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source><ph id="ph1">&lt;see langword="true" /&gt;</ph> if the exception should be thrown; otherwise, <ph id="ph2">&lt;see langword="false" /&gt;</ph>.</source>
          <target state="translated"><ph id="ph1">&lt;see langword="true" /&gt;</ph> Se a exceção deve ser gerada; Caso contrário, <ph id="ph2">&lt;see langword="false" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>The default is <ph id="ph1">&lt;see langword="false" /&gt;</ph>.</source>
          <target state="translated">O padrão é <ph id="ph1">&lt;see langword="false" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>Set this property to <ph id="ph1">`false`</ph> if the <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventHandler&gt;</ph> has dealt with the exception represented by the <ph id="ph3">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A&gt;</ph> property, and you do not want to propagate the exception to another <ph id="ph4">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventHandler&gt;</ph> that may handle the <ph id="ph5">&lt;xref:System.Windows.Forms.DataGridView.DataError&gt;</ph> event.</source>
          <target state="translated">Defina essa propriedade como <ph id="ph1">`false`</ph> se o <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventHandler&gt;</ph> tratou com a exceção representada pelo <ph id="ph3">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A&gt;</ph> propriedade e você não quiser propagar a exceção para outro <ph id="ph4">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventHandler&gt;</ph> que pode lidar com o <ph id="ph5">&lt;xref:System.Windows.Forms.DataGridView.DataError&gt;</ph> evento.</target>       </trans-unit>
        <trans-unit id="132" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>For debugging purposes, it can be useful to propagate the exception and inspect the stack trace for information regarding the context of the error.</source>
          <target state="translated">Para fins de depuração, ele pode ser útil propagar a exceção e inspecione o rastreamento de pilha para obter informações sobre o contexto do erro.</target>       </trans-unit>
        <trans-unit id="133" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>The following code example demonstrates using the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A&gt;</ph> property to indicate that the exception should not be thrown.</source>
          <target state="translated">O exemplo de código a seguir demonstra como usar o <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A&gt;</ph> propriedade para indicar que a exceção não deve ser gerada.</target>       </trans-unit>
        <trans-unit id="134" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>This example is part of a larger example available in the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> class overview topic.</source>
          <target state="translated">Este exemplo é parte de um exemplo maior disponível no <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> tópico de visão geral da classe.</target>       </trans-unit>
        <trans-unit id="135" translate="yes" xml:space="preserve" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>When setting this property to <ph id="ph1">&lt;see langword="true" /&gt;</ph>, the <ph id="ph2">&lt;see cref="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception" /&gt;</ph> property value is <ph id="ph3">&lt;see langword="null" /&gt;</ph>.</source>
          <target state="translated">Ao definir essa propriedade como <ph id="ph1">&lt;see langword="true" /&gt;</ph>, o valor da propriedade <ph id="ph2">&lt;see cref="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception" /&gt;</ph> é <ph id="ph3">&lt;see langword="null" /&gt;</ph>.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>